src/txn/txn_rollback_to_stable.c

```diff
@@ -156,6 +158,16 @@ __txn_abort_row_replace_with_hs_value(
 
     WT_ERR(__wt_row_leaf_key(session, page, rip, key, false));
 
+    /* Get the full update value from the data store. */
+    unpack = &_unpack;
+    __wt_row_leaf_value_cell(session, page, rip, NULL, unpack);
+    WT_CLEAR(ds_value);
+    if (!__wt_row_leaf_value(page, rip, &ds_value))
+        WT_ERR(__wt_page_cell_data_ref(session, page, unpack, &ds_value));
+
+    WT_CLEAR(full_value);
+    WT_ERR(__wt_buf_set(session, &full_value, ds_value.data, ds_value.size));
```

Comment:
I'm a little confused by the copy into `full_value`? What's the error it causes to use a single `WT_ITEM`?

Reply:
@keithbostic The copy here is added due to the same reasoning as: https://github.com/wiredtiger/wiredtiger/pull/5186#discussion_r377959764. As far as I can tell, it's the same issue we're trying to mitigate so the `memcpy` is still needed.

Reply:
You can `buf_set` into the current `WT_ITEM`.

Reply:
You mean like this?
```
WT_ERR(__wt_buf_set(session, &ds_value, ds_value.data, ds_value.size));
```
That's interesting, I hadn't thought of that.

Reply:
Yes.

Reply:
Thanks, I didn't realise you could do that! I made the change to just use the one `WT_ITEM` variable.